pos_default_quantity/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# 	    Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Set Default Product Quantity in POS",
+    "version": "12.0.1.0.0",
+    "author": "Coop IT Easy SCRLfs, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
```

Comment:
c'est notre module il n'est pas sur l'OCA du coup je propose de mettre https://coopiteasy.be

Reply:
website changed!